src/js/view/controls/components/timeslider.js

```diff
@@ -177,7 +177,7 @@ class TimeSlider extends Slider {
             return;
         }
         this.reset();
-        this.updateCues(model, model.get('cues'));
+        this.updateCues(model, []);
```

Comment:
would it be easier to get rid of the logic in `updateCues` and call `resetCues` here?

Reply:
`resetCues()` doesn't clear the cues on the model. I tried doing it but the build kept on failing with maximum stack call errors.

Reply:
oh duh, it gets called right after your change - my bad

Reply:
No what you want to do is get rid of the logic `updateCues` and update the model here. `updateCues` is fired when the model is changed.